src/App.tsx

```diff
@@ -561,10 +561,10 @@ function AppContent() {
 
 export default function App(): JSX.Element {
   return (
-    <AppBootstrap>
-      <Provider store={store}>
+    <Provider store={store}>
```

Comment:
Let's test and make sure there are no side effects from this?

Reply:
Looks like there are not